extensions/wikia/MyHome/SpecialWikiActivity.class.php

```diff
@@ -9,150 +9,147 @@ class SpecialWikiActivity extends UnlistedSpecialPage {
+		$feedProvider = new DataFeedProvider( $feedProxy );
 
 		global $wgJsMimeType, $wgExtensionsPath;
-		$wgOut->addScript("<script type=\"{$wgJsMimeType}\" src=\"{$wgExtensionsPath}/wikia/MyHome/WikiActivity.js\"></script>\n");
-		$wgOut->addExtensionStyle(AssetsManager::getInstance()->getSassCommonURL('extensions/wikia/MyHome/oasis.scss'));
+		$wgOut->addScript( "<script type=\"{$wgJsMimeType}\" src=\"{$wgExtensionsPath}/wikia/MyHome/WikiActivity.js\"></script>\n" );
+		$wgOut->addExtensionStyle( AssetsManager::getInstance()->getSassCommonURL( 'extensions/wikia/MyHome/oasis.scss' ) );
 
-		wfRunHooks( 'SpecialWikiActivityExecute', array( $wgOut, $wgUser ));
+		wfRunHooks( 'SpecialWikiActivityExecute', [ $wgOut, $wgUser ] );
 
-		$data = $feedProvider->get(50);  // this breaks when set to 60...
+		$data = $feedProvider->get( 50 );  // this breaks when set to 60...
 
 		// FIXME: do it in AchievementsII extension
 		global $wgEnableAchievementsInActivityFeed, $wgEnableAchievementsExt;
-		if((!empty($wgEnableAchievementsInActivityFeed)) && (!empty($wgEnableAchievementsExt))){
-			$wgOut->addExtensionStyle("{$wgExtensionsPath}/wikia/AchievementsII/css/achievements_sidebar.css");
+		if ( ( !empty( $wgEnableAchievementsInActivityFeed ) ) && ( !empty( $wgEnableAchievementsExt ) ) ) {
+			$wgOut->addExtensionStyle( "{$wgExtensionsPath}/wikia/AchievementsII/css/achievements_sidebar.css" );
 		}
 
 		// use message from MyHome as special page title
-		$wgOut->setPageTitle(wfMsg('oasis-activity-header'));
+		$wgOut->setPageTitle( wfMsg( 'oasis-activity-header' ) );
 
-		$template = new EasyTemplate(dirname(__FILE__).'/templates');
-		$template->set('data', $data['results']);
+		$template = new EasyTemplate( dirname( __FILE__ ) . '/templates' );
+		$template->set( 'data', $data['results'] );
 
-		$showMore = isset($data['query-continue']);
-		if ($showMore) {
-			$template->set('query_continue', $data['query-continue']);
+		$showMore = isset( $data['query-continue'] );
+		if ( $showMore ) {
+			$template->set( 'query_continue', $data['query-continue'] );
 		}
-		if (empty($data['results'])) {
-			$template->set('emptyMessage', wfMsgExt("myhome-activity-feed-empty", array( 'parse' )));
+		if ( empty( $data['results'] ) ) {
+			$template->set( 'emptyMessage', wfMsgExt( "myhome-activity-feed-empty", [ 'parse' ] ) );
 		}
 
-		$template->set_vars(array(
+		$template->set_vars( [
 			'showMore' => $showMore,
 			'type' => $this->feedSelected,
 			'wgBlankImgUrl' => $wgBlankImgUrl,
-		));
+		] );
 
-		$wgOut->addHTML($template->render('activityfeed.oasis'));
+		$wgOut->addHTML( $template->render( 'activityfeed.oasis' ) );
 
 		// page header: replace subtitle with navigation
 		global $wgHooks;
-		$wgHooks['PageHeaderIndexAfterExecute'][] = array($this, 'addNavigation');
+		$wgHooks['BeforePageHeaderSubtitle'][] = [ $this, 'addNavigation' ];
 
-		if ($wgUser->isAnon()) {
+		if ( $wgUser->isAnon() ) {
 			$this->getOutput()->setSquidMaxage( 3600 ); // 1 hour
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)